sample-xml/buildSrc/src/main/java/plugins/jacoco-report.gradle.kts

```diff
@@ -0,0 +1,142 @@
+package plugins
+
+plugins {
+    jacoco
+}
+
+jacoco {
+    toolVersion = Versions.JACOCO_VERSION
+}
+
+val fileGenerated = setOf(
+    "**/R.class",
+    "**/R\$*.class",
+    "**/*\$ViewBinder*.*",
+    "**/*\$InjectAdapter*.*",
+    "**/*Injector*.*",
+    "**/BuildConfig.*",
+    "**/Manifest*.*",
+    "**/*_ViewBinding*.*",
+    "**/*Adapter*.*",
+    "**/*Test*.*",
+    // Enum
+    "**/*\$Creator*",
+    // Nav Component
+    "**/*_Factory*",
+    "**/*FragmentArgs*",
+    "**/*FragmentDirections*",
+    "**/FragmentNavArgsLazy.kt",
+    "**/*Fragment*navArgs*",
+    "**/*ModuleDeps*.*",
+    "**/*NavGraphDirections*",
+    // Hilt
+    "**/*_ComponentTreeDeps*",
+    "**/*_HiltComponents*",
+    "**/*_HiltModules*",
+    "**/Hilt_*"
+)
+
+val packagesExcluded = setOf(
+    "**/com/bumptech/glide",
+    "**/dagger/hilt/internal",
+    "**/hilt_aggregated_deps",
+    "**/co/nimblehq/template/databinding/**",
+    "**/co/nimblehq/template/di/**"
```

Comment:
```suggestion
    "**/co/nimblehq/sample/xml/databinding/**",
    "**/co/nimblehq/sample/xml/di/**"
```

Reply:
@Tuubz How about we have the script to manage to change the path once we generate a new project too? 🤔

Reply:
@lydiasama I've created an issue for this: https://github.com/nimblehq/android-templates/issues/275 🚀